tests/tests/test-precompile/test-precompile-lowlevel-txns.ts

```diff
@@ -0,0 +1,175 @@
+import "@moonbeam-network/api-augment";
+
+import { u128 } from "@polkadot/types";
+import { BN, bnToHex, numberToHex, stringToHex } from "@polkadot/util";
+import { expect } from "chai";
+import { ethers } from "ethers";
+
+import { alith, ALITH_ADDRESS, baltathar, charleth } from "../../util/accounts";
+import { mockAssetBalance } from "../../util/assets";
+import { getCompiled } from "../../util/contracts";
+import { web3EthCall } from "../../util/providers";
+import { describeDevMoonbeamAllEthTxTypes } from "../../util/setup-dev-tests";
+import type { PalletAssetsAssetAccount, PalletAssetsAssetDetails } from "@polkadot/types/lookup";
+
+import {
+  ALITH_TRANSACTION_TEMPLATE,
+  BALTATHAR_TRANSACTION_TEMPLATE,
+  CHARLETH_TRANSACTION_TEMPLATE,
+  createContract,
+  createTransaction,
+} from "../../util/transactions";
+import { createImportSpecifier } from "typescript";
+
+const ADDRESS_ERC20 = "0xFfFFfFff1FcaCBd218EDc0EbA20Fc2308C778080";
+const ASSET_ID = new BN("42259045809535163221576417993425387648");
+const SELECTORS = {
```

Comment:
We dont need these I think, we can remove them from both files

Reply:
done